about.php

```diff
@@ -12,59 +12,46 @@
 
 <body>
 	<?php include 'navbar.php'; ?>
-	<h2 class="us">About Us</h2>
-	<div class="company">
-		<h3>Company Information</h3>
-		<p>We are the StarBulls. The mighty, mighty StarBulls. Everywhere we go, people want to know:
-			WHO WE
-			ARE! So we tell
-			them: WE ARE THE STARBULLS; THE MIGHTY, MIGHTY STARBULLS!</p>
-	</div>
-	<div class="heritage">
-		<h3>Our Heritage</h3>
-		<p>We are humbled by the success of our company thus far. The name StarBulls was created by Will, 
-			one of our founders. Fueled by a continual flow of caffeine and blue cheese, Will opened StarBulls in Febuary of 2021.
-		</p>
-		<p>How do we measure success? Simply: we started from the bottom, now we are here! Seriously,
-			look at our site! A beauty to say the least.
-		</p>
-		<p>Just a week ago, StarBulls did not exist. It has taken hours of dedication, caffeine, and chicken wings to get this 
-			company up on the server. Days of frustration, trying to figure out why the code looks different on
-			our screens. Hours of trial and error as we figured out where we went wrong. One thing the
-			founders of StarBulls knew from the beginning is: Quitters Never Win. And Winners Never Quit. At
-			StarBulls we are Winners!
-		</p>
-	</div>
-	<div class="tools">
-		<h3>Tools We Used</h3>
-		<p>Like any other company, the founders of StarBulls rely on various resources and tools to help
-			create this site. We want to share our tools and resources with the world!
-		</p>
-		<h3>They are:</h3>
-		<ol>
-			<li>Bill C.: You may have heard of him. The Man. The Myth. The Legend. If you haven't, then
-				maybe StarBulls is not for you!</li>
-			<li>Google.com: This one can have you running in circles BUT a lot of questions have been
-				answered by our friend Google.</li>
-			<li>Last but not least, the founders at StarBulls, relied on one another and other
-				classmates.
-			</li>
-		</ol>
-	</div>
-	<div id="myCarousel" class="carousel slide" data-ride="carousel">
-		<div class="carousel-inner">
-			<img src="imgs/cofee3.png" width="100%">
-			<div class="item active">
-				<div class="carousel-caption d-none d-md-block">
-					<div class="hero-text">
-						<h2>StarBulls</h2>
+	<div class="conatiner containerAbout">
+		<div id="myCarousel" class="carousel slide" data-ride="carousel">
+			<div class="carousel-inner">
+				<img src="imgs/cofee3.png" width="100%">
+				<div class="item active">
+					<div class="carousel-caption">
+						<h1 class="p-4">About Us</h1>
```

Comment:
the h1 for About Us need the class hero-text to pick up the styling used on the Home Page and Apply Now pages.

Reply:
Ok, sounds good